.github/workflows/wheels.yml

```diff
@@ -109,7 +109,7 @@ jobs:
       - name: setup rtools for 32-bit
         run: |
           echo "PLAT=i686" >> $env:GITHUB_ENV
-          echo "PATH=$env:RTOOLS40_HOME\mingw32\bin;$env:PATH" >> $env:GITHUB_ENV
+          echo "PATH=$env:RTOOLS43_HOME\mingw32\bin;$env:PATH" >> $env:GITHUB_ENV
```

Comment:
I don't think you want this. RTOOLS43_HOME would be for the one that's just been released.

On scipy we use:
```
          choco install rtools -y --no-progress --force --version=4.0.0.20220206
          echo "c:\rtools40\ucrt64\bin;" >> $env:GITHUB_PATH
```
that puts the 64 bit  gcc/gfortran/pkg-config on PATH. If you want the 32-bit tools they're in C:\rtools40\mingw32\bin. You can either put those directories on PATH or set the RTOOLS40_HOME variable correctly.

To fix the issue I think you need to set the RTOOLS40_HOME variable, i.e.

$env:RTOOLS40_HOME = "c:\rtools40"

or 

echo "RTOOLS40_HOME=c:\rtools40" >> "$GITHUB_ENV"